<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.dentsu.bootcamp.service.ReservationServiceTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2024-12-23T13:27:37" hostname="BR-FVFFH46JQ6LT" time="0.73">
  <properties/>
  <testcase name="givenInvalidDetails_whenGetReservation_thenThrowReservationNotFoundException()" classname="com.dentsu.bootcamp.service.ReservationServiceTest" time="0.73">
    <failure message="org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: &lt;com.dentsu.bootcamp.exception.ReservationNotFoundException&gt; but was: &lt;java.lang.NullPointerException&gt;" type="org.opentest4j.AssertionFailedError">org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: &lt;com.dentsu.bootcamp.exception.ReservationNotFoundException&gt; but was: &lt;java.lang.NullPointerException&gt;
	at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at app//org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
	at app//org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at app//org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at app//com.dentsu.bootcamp.service.ReservationServiceTest.givenInvalidDetails_whenGetReservation_thenThrowReservationNotFoundException(ReservationServiceTest.java:333)
	at java.base@17.0.13/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base@17.0.13/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base@17.0.13/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.NullPointerException: Callable returned a null value. Null values are generally not allowed in 3.x operators and sources.
	at io.reactivex.rxjava3.internal.util.ExceptionHelper.createNullPointerException(ExceptionHelper.java:164)
	at io.reactivex.rxjava3.internal.util.ExceptionHelper.nullCheck(ExceptionHelper.java:179)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableFromCallable.subscribeActual(ObservableFromCallable.java:46)
	at io.reactivex.rxjava3.core.Observable.subscribe(Observable.java:13263)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableDoOnEach.subscribeActual(ObservableDoOnEach.java:42)
	at io.reactivex.rxjava3.core.Observable.subscribe(Observable.java:13263)
	at io.reactivex.rxjava3.internal.operators.observable.ObservableMap.subscribeActual(ObservableMap.java:33)
	at io.reactivex.rxjava3.core.Observable.subscribe(Observable.java:13263)
	at io.reactivex.rxjava3.core.Observable.blockingFirst(Observable.java:5453)
	at com.dentsu.bootcamp.service.ReservationServiceTest.lambda$givenInvalidDetails_whenGetReservation_thenThrowReservationNotFoundException$4(ReservationServiceTest.java:334)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)
	... 6 more
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
